src/unistd.rs

```diff
@@ -1398,6 +1398,30 @@ pub fn setgid(gid: Gid) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Set the user identity used for filesystem checks.
```

Comment:
Perhaps add a "for the current thread" here?  I found it non-obvious that this is a per-thread variable.

Reply:
Done.